codec.go

```diff
@@ -32,6 +32,12 @@ type InterfaceInfo struct {
 	InterfaceOptions
 }
 
+func (ti *TypeInfo) disfix() DisfixBytes {
```

Comment:
Can we remove this and just call toDisfix(...)?  I'm trying to leave out methods from these structs.  If we need to, it would make more sense to add it to `type ConcreteInfo struct{}`, but overall it's adding more lines of code than not...